eras/conway/impl/src/Cardano/Ledger/Conway/Governance.hs

```diff
@@ -677,10 +683,16 @@ votingCommitteeThresholdInternal committee = \case
   TreasuryWithdrawals {} -> threshold
   InfoAction {} -> NoVotingThreshold
   where
-    threshold =
-      case committeeQuorum <$> committee of
-        SJust t -> VotingThreshold t
-        SNothing -> NoVotingThreshold
+    threshold
+      -- if the committee size is smaller than the mnimimum given in pparams,
+      -- we treat it as if we had no committe
+      | committeeSize < minSize = NoVotingThreshold
+      | otherwise =
+          case committeeQuorum <$> committee of
+            SJust t -> VotingThreshold t
+            SNothing -> NoVotingThreshold
+    minSize = fromIntegral $ pp ^. ppCommitteeMinSizeL
+    committeeSize = Map.size $ foldMap' committeeMembers committee
```

Comment:
Now this is wrong.

`NoVotingThreshold` means that action will be accepted.

This is a wrong place to handle this case of committee too small

Reply:
Yes, I wasn't sure about this at all actually

Reply:
Ignore my comment, you are totally right!!!! :smile: